docs/book/src/user/quick-start.md

```diff
@@ -1015,6 +1105,23 @@ clusterctl generate cluster ${CLUSTER_NAME} \
     --target-namespace ${CLUSTER_NAMESPACE} | kubectl apply -f -
 ```
 
+{{#/tab }}
+{{#tab KubeVirt}}
+
+Create the workload cluster using the load balancer template  
+```bash
+CAPK_VER=$(curl "https://api.github.com/repos/kubernetes-sigs/cluster-api-provider-kubevirt/releases/latest" | jq -r ."tag_name")
+
+curl -L -O "https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/releases/download/${CAPK_VER}/cluster-template-lb.yaml"
+
+clusterctl generate cluster capi-quickstart \
+  --kubernetes-version ${CAPK_GUEST_K8S_VERSION} \
+  --control-plane-machine-count=1 \
+  --worker-machine-count=1 \
+  --from cluster-template-lb.yaml \
+  > capi-quickstart.yaml
```

Comment:
```suggestion
clusterctl generate cluster capi-quickstart --flavor lb \
  --kubernetes-version ${CAPK_GUEST_K8S_VERSION} \
  --control-plane-machine-count=1 \
  --worker-machine-count=1 \
  --from cluster-template-lb.yaml \
  > capi-quickstart.yaml
```

Should work now

Reply:
:+1: :+1: :+1: 

These are good news. I opened a related issue (#7455) + PR to fix it (#7453), in order to use the `--form` flag to get a template from release assets. I still think it should be merged, but in this case, the `--flavor` flag will work.

Reply:
Sounds good to me!

Reply:
@sbueringer - done.

Reply:
Thx!